src/app/components/Metadata/index.tsx

```diff
@@ -79,6 +80,9 @@ const getSocialShareImage = ({
   pathname: string;
   service: Services;
 }) => {
+  // TODO: Remove to release experiment
```

Comment:
Shall we alter the comment to show we have done the experiment but it is ready to release to production next when removing the isLive?

Could be something like Remove to release to live ot Remove to release to production. Just so people looking know we have already done the experiment.

Unless we are going to do the experiment again on other service?

Otherwise all good.